tests/python-e2e/ops.py

```diff
@@ -18,6 +18,49 @@ def test_read_write_family(self):
             self.assertEqual(read_text, TEXT)
             rw_file.close()
 
+    def test_fopen(self):
+        """
+        Opens a file in "/tmp/"
+        """
+        file_path, _ = self._create_new_tmp_file()
+        self.assertFalse(self._check_path_exists_on_host(file_path))
+        with open(file_path, "r") as f:
+            f.seek(0)
+            txt = f.readline()
+        self.assertEqual(txt, TEXT)
+
+    def test_fdopen(self):
+        """
+        Opens a file in "/tmp"
+        """
+        file_path, _ = self._create_new_tmp_file()
+        self.assertFalse(self._check_path_exists_on_host(file_path))
+        fd = os.open(file_path, os.O_RDONLY)
+        self.assertEqual(fd, 3)
+        fd.close()
+
+    def test_fread(self):
```

Comment:
hmm we are not testing any file reads here, we should assert some kind of text read